script/bootstrap

```diff
@@ -520,6 +588,63 @@ function bootstrapOnRedHat ()
                                 gcc-c++ \
                                 openal-soft-devel \
                                 SDL2-devel \
+                                SDL2_image-devel \
+                                libvorbis-devel \
+                                libglvnd-devel \
+                                libjpeg-turbo-devel \
+                                libpng-devel \
+                                expat-devel \
+                                gtk3-devel \
+                                python3-devel \
+                                libarchive-devel \
+                                rpm-build \
+                                make \
+                                clang
+            ;;
+        "9.6")
```

Comment:
It looks like 9.6 and 10.0 are duplicated three times.

Reply:
@royfalk for CentOS, Red Hat, and Rocky Linux - so yes, that would make sense.

Reply:
Ah. Missed that.